keps/sig-scheduling/5055-dra-device-taints-and-tolerations/README.md

```diff
@@ -229,6 +259,24 @@ the purpose of this KEP:
 - The ResourceClaim controller will remove such a completed pod from the claim's
   `ReservedFor` and deallocate the claim once it has no consumers.
 
+The semantic of the value associated with a taint key is defined by whoever
+publishes taints with that key. DRA drivers should use the driver name as
+domain in the key to avoid conflicts. To support tolerating taints by value,
+values should not require parsing to extract information. It's better to use
+different keys with simple values than one key with a complex value.
+
+The taint value may or may not be sufficient to represent the state of a
+device. Therefore the API also allows publishing structured information in JSON
+format as a raw extension field, similar to the ResourceClaim device status.
+For humans, a description field can be used to provide a summary or additional
+explanations why the taint was added.
+
+If some common patterns emerge, then Kubernetes could standardize the name,
+value and data for certain taints. Those then have a certain semantic. This
+is similar to standardizing device attributes.
+
+`Effect: None` can be used to publish taints which are merely informational.
+
```

Comment:
How cluster admins can constrain tenant tolerations for device taints to block tolerating `NoExecute` for safety-critical taints?

We may need to describe how the multi-tenant controls is being handled. The cluster admins may want to prevent users from tolerating certain device taints (e.g., `NoExecute` on unsafe devices.

Reply:
> Do we have such a feature for node taints? 
No, we don't, the scheduler just assumes tolerations in the PodSpec are allowed.

But it would be useful if the KEP calls it out explicitly in Risks and Mitigations. Otherwise readers may assume toleration scoping is somehow solved by this design when in fact it relies on existing policy mechanisms.